### PR TITLE
[DOP-22268] - move spark logs to json format

### PR DESCRIPTION
**Pull request by maxim-lixakov**
## Change Summary

previously logs of spark process in worker was in plain format:

```
2024-12-10 18:46:55,827 [INFO    ] MainThread: Using mtspark 4.4.1
2024-12-10 18:46:55,827 [INFO    ] MainThread: Current user is 'syncmaster'
```
now they are formated to `json` and consist `correlation_id` from initial request:

```
2024-12-17 16:41:39 {"processName": "ForkPoolWorker-8", "process": 15, "threadName": "MainThread", "thread": 281472838234144, "name": "onetl.connection.db_connection.jdbc_mixin.connection", "lineno": 148, "levelname": "INFO", "message": "|Postgres| Checking connection availability...", "correlation_id": "0193d4da440f7e1c869a044bf75b1fdc", "timestamp": "2024-12-17T13:41:39.741075+00:00"}
2024-12-17 16:41:39 {"processName": "ForkPoolWorker-8", "process": 15, "threadName": "MainThread", "thread": 281472838234144, "name": "onetl.connection.db_connection.db_connection.connection", "lineno": 56, "levelname": "INFO", "message": "|Postgres| Using connection parameters:", "correlation_id": "0193d4da440f7e1c869a044bf75b1fdc", "timestamp": "2024-12-17T13:41:39.741142+00:00"}
```
## Checklist

* [x] Commit message and PR title is comprehensive
* [x] Keep the change as small as possible
* [ ] Unit and integration tests for the changes exist
* [ ] Tests pass on CI and coverage does not decrease
* [ ] Documentation reflects the changes where applicable
* [ ] `docs/changelog/next_release/<pull request or issue id>.<change type>.rst` file added describing change
  (see [CONTRIBUTING.rst](https://github.com/MobileTeleSystems/syncmaster/blob/develop/CONTRIBUTING.rst) for details.)
* [x] My PR is ready to review.

### CODE REVIEW

File: syncmaster/worker/transfer.py [hunk 1]
```diff
@@ -20,14 +22,15 @@
 from syncmaster.worker.settings import WorkerAppSettings
 
 logger = get_task_logger(__name__)
+load_correlation_ids()
+load_celery_current_and_parent_ids()
```
Comment (dolfinus) [inline syncmaster/worker/transfer.py:26]: Shouldn't we call these in after_setup_task_logger or before_setup_task_logger?

Comment (maxim-lixakov) [reply syncmaster/worker/transfer.py]: no, we should call it globally when starting celery app, then any other hooks will be able to retrieve correlation id from headers and set it in celery tasks

Comment (dolfinus) [reply syncmaster/worker/transfer.py]: I see, this call actually creates its own hooks:
https://github.com/snok/asgi-correlation-id/blob/ed006dcc119447bf68a170bd1557f6015427213d/asgi_correlation_id/extensions/celery.py#L14

Do we actually need `transfer_correlation_id` and `load_correlation_id` then?

Comment (maxim-lixakov) [reply syncmaster/worker/transfer.py]: we do not need, thereby we could also remove extra worker setting: `CORRELATION_CELERY_HEADER_ID `

**Review by dolfinus — APPROVED** [review]

**Pull request merged**